packages/ui/src/app/pages/WorkingGroups/MyRoles/MyRole.tsx

```diff
@@ -110,9 +110,9 @@ export const MyRole = () => {
         </Statistics>
         <ContentWithSidepanel>
           <MainPanel>
-            <MyRoleAccount name="Role Account" address={worker.roleAccount} />
-            <MyRoleAccount name="Staking Account" address={worker.stakeAccount} />
-            <MyRoleAccount name="Reward Account" address={worker.rewardAccount} />
+            <MyRoleAccount account={{ name: 'Role Account', address: worker.roleAccount }} canChangeAccount={true} />
```

Comment:
```suggestion
           <RoleAccountHeader>
             <Label>Role account</Label>
             {isActive && ( <ButtonsGroup>
                 <ButtonGhost size="small">Change Role account</ButtonGhost>}
             </ButtonsGroup> ) }
          </RoleAccountHeader>
           <MyRoleAccount name="Staking Account" address={worker.stakeAccount} />
```

Something like, above might be better to compose the page instead of making the `<MyRoleAccount />` aware of different actions that it could invoke.

Reply:
OMG, I can't properly indent my comment above :roll\_eyes: But the idea is to make the MyRole account unaware of those buttons and compose the page using the components.

I've added `isActive` there to indicate that these buttons should be visible only when the worker is active.

ps.: The button can be also disabled in the unstaking period, but we don't have that one yet.